pmd-cpp/etc/grammar/cpp.jj

```diff
@@ -312,9 +312,69 @@ TOKEN :
 
   <  CHARACTER : ("L")? "'" ( ( ~["'","\\","\r","\n"] ) | ( "\\" ( ~["\n","\r"] ) ) )* "'" >
 
-| <  STRING : ("L")? "\"" ( ( ~["\"","\\","\r","\n"] ) | ( "\\" ( ~["\n","\r"] | "\n" | "\r\n" ) ) )* "\"" >
+| < #STRPREF : (["L", "u", "U"] | "u8")? >
```

Comment:
I'd write this without additional nesting:

```suggestion
| < #STRPREF : ("L" | "u" | "U" | "u8")? >
```

I think, it's a little bit easier to read.

I was looking at the character token. According to https://en.cppreference.com/w/cpp/language/character_literal the same prefixes are used of characters as well. Maybe we should cover it in this PR?